### ISSUE TITLE
Enable writing kern values (GPOS)

### ISSUE THREAD

**Issue by mattlag** [glyphr-studio/Glyphr-Studio-2#228]

**Comment by mattlag** [comment]
Hi @Balearica

I'm working with opentype.js and I would absolutely love to be able to write even a basic GPOS table. I saw you mentioned in your comment below that you did this. Any information or pointers would be amazing! If it's solid enough, I may also turn it into a pull request... but at this point I'd go with a hacky solution that I use locally.

https://github.com/opentypejs/opentype.js/issues/112#issuecomment-1087151544

Thank you!

**Comment by Balearica** [comment]
@mattlag As noted in the comment linked, I created a fork of OpenType.js that supports GPOS type 2 lookups (kerning), which is used for generating fonts in my document conversion web application [scribeocr.com](https://scribeocr.com/) (repo [here](https://github.com/scribeocr/scribeocr)).  Since writing that I cleaned up the code and created the merge request opentypejs/opentype.js#743.  While it has not yet been merged, that fork should be ready to use.
